InventoryDistanceBasedSourceSelection/Plugin/FillSourceLatitudeAndLongitude.php

```diff
@@ -53,8 +71,12 @@ public function beforeSave(
 
                 $source->setLatitude($latLng->getLat());
                 $source->setLongitude($latLng->getLng());
-            } catch (\Exception $e) {
-                unset($e); // Silently fail geo coding
+            } catch (LocalizedException $exception) {
+                $this->logger->error($exception);
```

Comment:
What about `LocalizedException` logging removal? It is shown to the user "as is" anyway.